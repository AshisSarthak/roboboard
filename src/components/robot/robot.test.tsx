import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Robot from ".";

test("rendersRobotBox", () => {
  const { asFragment } = render(<Robot />);
  expect(screen.getByTestId("robot-box")).toBeInTheDocument();
  expect(asFragment()).toMatchSnapshot();
});
