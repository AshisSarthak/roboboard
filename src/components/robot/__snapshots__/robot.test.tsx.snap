// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendersRobotBox 1`] = `
<DocumentFragment>
  <div
    class="direction-arrow robot MuiBox-root css-v399b5"
    data-testid="robot-box"
  >
    <img
      alt=""
      class="north"
      src="robot-vacuum-cleaner.png"
    />
  </div>
</DocumentFragment>
`;
